products/memcache/api.yaml

```diff
@@ -92,6 +93,35 @@ objects:
             - !ruby/object:Api::Type::String
               name: 'message'
               description: The message to be displayed to a user.
+      - !ruby/object:Api::Type::Array
+        name: 'memcacheNodes'
+        description: |
+          Additional information about the instance state, if available.
+        output: true
```

Comment:
You'll also need to add `output: true` for each of the nested fields (see https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-3821-old..auto-pr-3821#diff-27973e511911c94f8104f416cb8733abR157-R192 for what it ends up looking like, with only the outer one being Computed and the rest being Optional).